src/components/App.js

```diff
@@ -16,25 +19,100 @@ import '../style.css';
 		*Provide form to input repo information dynamically, and display related issues
 */
 
+const options = [
+    {
+      label: "All",
+      value: "all",
+      size: "80px"
+    },
+    {
+      label: "Open",
+      value: "open",
+      size: "100px"
+    },
+    {
+      label: "Closed",
+      value: "closed",
+      size: "110px"
+    },
+  ];
+
 const App = () => {
     
     const [issue, setIssue] = useState([]);
+    const [label, setLabel] = useState([]);
+    const [pageNumber, setpageNumber] = useState(0);
+    const [githubState, setgithubState] = useState(options[0].value); // for filter open/closed/all
+
+    const loadIssues = async (pageNumber) => {
+        let gitState = '';
+            if(githubState === 'all') {
+                gitState = '&state=all'
+            } else {
+                gitState = '&state=' + githubState.value
+            }
 
-        const loadIssues = async () => {
-            const response = await github.get();
+        if (pageNumber === 0) {
+            setpageNumber(1)
+            const response = await github.get('issues?page=' + pageNumber +gitState);
```

Comment:
```suggestion
            const response = await github.get('/issues?page=' + pageNumber +gitState);
```
is it not `/issues...`? is it working with your current implementation? did you test it?

Reply:
yes sir it's working sir.